src/js/letters/actions/letters.js

```diff
@@ -165,6 +160,9 @@ export function getLetterPdf(letterType, letterName, letterOptions) {
               save.click();
               document.body.removeChild(save);
             } else {
+              // Instead of giving the file a readable name and downloading
+              // it directly, open it in a new window with an ugly hash URL
+              const downloadWindow = window.open();
```

Comment:
Previously, we'd set `ie10` to true _and_ `downloadSupported` to false, so we'd open a window and do nothing with it.